Peralta/template/simple_functions.py

```diff
@@ -10,3 +10,19 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
+def is_prime(value):
+    assert type(value) is int
+    
+    # Initialise the result as True
+    result = True
+
+    # Try to find a integer which divides the input value:
+    # If we find one, the result will be changed to False
+    for i in range(2, value):
```

Comment:
I think n<=1 needs to be checked. It seems like, this function will return True if the input is, e.g., 1.

Reply:
You are completely right: now I will check that cases too.